src/motion/features/layout/Measure.tsx

```diff
@@ -6,10 +6,12 @@ import {
     SharedLayoutSyncMethods,
     isSharedLayout,
     SharedLayoutContext,
+    FramerTreeContext,
 } from "../../../components/AnimateSharedLayout/SharedLayoutContext"
 
 interface SyncProps extends FeatureProps {
     syncLayout: SharedLayoutSyncMethods | SyncLayoutBatcher
+    framerTreeSync: SharedLayoutSyncMethods | SyncLayoutBatcher
```

Comment:
```suggestion
    syncTreeLayout: SharedLayoutSyncMethods | SyncLayoutBatcher
```

Reply:
I'd prefer this to mirror the context name and for it to be Framer-specific